image-counter-lambda/package.json

```diff
@@ -13,15 +13,15 @@
   "license": "ISC",
   "devDependencies": {
     "@guardian/node-riffraff-artifact": "^0.1.5",
-    "@types/jest": "^24.9.0",
+    "@types/jest": "^29.2.1",
     "@types/node": "^13.1.8",
-    "@zeit/ncc": "^0.21.0",
-    "jest": "^24.9.0",
-    "ts-jest": "^24.3.0",
-    "typescript": "^3.7.5"
+    "@vercel/ncc": "^0.34.0",
```

Comment:
not for this PR, but we could probs move to `esbuild` - much faster (see pinboard for examples)